CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     [#795](https://github.com/GetStream/stream-chat-swift/pull/795)
 - Add possibility to send attachments that don't need prior uploading
     [#799](https://github.com/GetStream/stream-chat-swift/pull/799)
+- Add MacOS support for LLC
+    [#811](https://github.com/GetStream/stream-chat-swift/pull/811)
```

Comment:
nit 1: it's `macOS` ;-)
nit 2: We call it "LLC" just internally, the official name is `StreamChat`. However, since everything here is about `StreamChat`, you can just say something like "Add possibility to build the framework on `macOS`"